### PR TITLE
DOCS-5432 add section about oauth access token

### PR DESCRIPTION
**Pull request by alexisintech**
you can also test the example yourself: https://github.com/alexisintech/clerk-nextjs-core1/blob/notion/app/api/notion/route.tsx

### CODE REVIEW

File: docs/authentication/social-connections/oauth.mdx [hunk 1]
```diff
@@ -59,15 +50,13 @@ With Clerk, you can request additional OAuth scopes even after a user has signed
 
 For [Clerk Components](/docs/components/overview), this is handled automatically by Clerk. You only have to pass the [`additionalOAuthScopes`](/docs/components/user/user-profile) prop in the [`<UserProfile/>`](/docs/components/user/user-profile) component or the corresponding [`userProfileProp`](/docs/components/user/user-button) in the [`<UserButton />`](/docs/components/user/user-button) component, with any additional OAuth scope you would like per provider.
 
-## OAuth Access Token Wallet
+## Get an OAuth access token for a social login provider
 
-You can retrieve the OAuth access tokens of your users via the [OAuth Access Token Wallet](https://clerk.com/docs/reference/backend-api/tag/Users#operation/GetOAuthAccessToken) endpoint that's available in the Clerk Backend API.
+To query an OAuth progvider for additional data of your users, you can use the OAuth access token.
```
Comment (jescalan) [inline docs/authentication/social-connections/oauth.mdx:55]:
```suggestion
You can use a provider's OAuth access token to access user data from that provider in addition to their data from Clerk.
```

File: docs/authentication/social-connections/oauth.mdx [hunk 2]
```diff
@@ -59,15 +50,13 @@ With Clerk, you can request additional OAuth scopes even after a user has signed
...
 
-Using these tokens, you can query the respective OAuth providers for additional data of your users.
+To retrieve the OAuth access token for a user, use the `getUserOauthAccessToken()` method from the Clerk Backend SDK. Note that this method must be used in a server environment, and cannot be run on the client.
```
Comment (jescalan) [inline docs/authentication/social-connections/oauth.mdx:57]: We should link to the page for it here

**Review by jescalan — CHANGES_REQUESTED** [review]
This is good! Just a couple comments

File: docs/authentication/social-connections/oauth.mdx [hunk 3]
```diff
@@ -7,14 +7,7 @@ description: Learn how to effortlessly add social connections to your applicatio
...
 
-<Images
-  width={1879}
-  height={1373}
-  src="/docs/images/setup-clerk/create-sign-in-comp.png"
-  alt="An interactive builder for customizing your <SignIn /> component, which includes choosing identifiers and social login options."
-/>
-
-Social connections provide better security than passwords and other long-lived knowledge-based secrets. With social connections, users gain frictionless access to your application by using their existing credentials from an OAuth provider (Google, Facebook, Twitter, etc.) without having to go through complicated registration flows.
+Social connections provide better security than passwords and other long-lived knowledge-based secrets. With social connections, users gain frictionless access to your application by using their existing credentials from an OAuth provider (Google, Facebook, X/Twitter, etc.) without having to go through complicated registration flows.
```
[the remark this answers is not in the document]
Comment (alexisintech) [reply docs/authentication/social-connections/oauth.mdx:10]: I have no clue who wrote this LMAO I can remove it though o.O

File: docs/authentication/social-connections/oauth.mdx (hunk 2 above)
Comment (alexisintech) [reply docs/authentication/social-connections/oauth.mdx:57]: oh I'm silly I forgot that

File: docs/authentication/social-connections/oauth.mdx [hunk 4]
```diff
@@ -59,15 +50,13 @@ With Clerk, you can request additional OAuth scopes even after a user has signed
...
-You can retrieve the OAuth access tokens of your users via the [OAuth Access Token Wallet](https://clerk.com/docs/reference/backend-api/tag/Users#operation/GetOAuthAccessToken) endpoint that's available in the Clerk Backend API.
+To query an OAuth progvider for additional data of your users, you can use the OAuth access token.
 
-Using these tokens, you can query the respective OAuth providers for additional data of your users.
+To retrieve the OAuth access token for a user, use the `getUserOauthAccessToken()` method from the Clerk Backend SDK. Note that this method must be used in a server environment, and cannot be run on the client.
 
-<Callout type="info">
-  Clerk ensures that the OAuth Access Token will be always fresh so that you don't have to worry about OAuth Refresh Tokens anymore.
-</Callout>
+Clerk ensures that the OAuth Access Token will be always fresh so that you don't have to worry about OAuth Refresh Tokens anymore.
```
[the remark this answers is not in the document]
Comment (alexisintech) [reply docs/authentication/social-connections/oauth.mdx:57]: added!

File: docs/authentication/social-connections/oauth.mdx [hunk 5]
```diff
@@ -59,15 +48,61 @@ With Clerk, you can request additional OAuth scopes even after a user has signed
...
+      "Notion-Version": "2022-06-28",
+    },
+  });
+
+  // Handle the response from the Notion API
+  const notionData = await notionResponse.json();
+
+  return NextResponse.json({ message: notionData });
+}
+```
```
Comment (jescalan) [inline docs/authentication/social-connections/oauth.mdx:105]: 😍

**Review by jescalan — APPROVED** [review]
This is 🌟

**Pull request merged**